skins/laika/src/components/pages/donation_form/subpages/AddressPage.vue

```diff
@@ -8,15 +8,15 @@
 				</donation-summary>
 				<submit-values :tracking-data="{}"></submit-values>
 				<div class="level has-margin-top-36">
-					<div class="level-left">
+					<div class="level-item">
```

Comment:
Is the `level` type the still the right one, if we no longer distinguish between right and left. Wouldn't the `column` classes be more appropriate?

Reply:
Yeah, makes sense.